study-builder/studies/pancan/study-events.conf

```diff
@@ -376,7 +376,7 @@
         ],
         "linkedActivityCode": ${id.act.release},
         "pdfAttachments": [
-          // { "pdfName": "countmein-consent" }
+           { "pdfName": "countmein-consent" }
```

Comment:
There should be one more consent email (for the age-up flow) that need to uncomment the pdf attachment.

Reply:
Is it the reconsent email or something else?
 ## In age-up flow, queue up reconsent thank you email (line# 469)

Reply:
Yup!